source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -509,18 +516,52 @@ It will feature two ACLs, with the following data:
 + ingress rule
   * priority: DROP_PRIORITY
   * action: drop
-  * direction: from-lport
+  * direction: to-lport
   * match: inport == @<port_group> && ip
 + egress rule
```

Comment:
Can you remind me why the `&& ip` is required?

Reply:
we want to mimic openstack's behavior, which means allowing L2 traffic. 

so we block everything on layers >= 3

Reply:
thanks